Question.py

```diff
@@ -0,0 +1,20 @@
+import kivy
+import json
+import random
+
+
+class Question():
+    def __init__(self, question_text, question_id, question_audio):
+        self.question_text = question_text
+        self.question_id = question_id
+        self.question_audio = question_audio
+        self.explanation_text = None
+        self.explanation_audio = None
```

Comment:
Let's add explanation text and audio to the Drag and Drop / Multiple Choice classes, and pass in those two fields here in this constructor